src/test/java/jalau/at18/Azul/BagTest.java

```diff
@@ -0,0 +1,36 @@
+package jalau.at18.Azul;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+import jalau.at18.azul.Bag;
+import jalau.at18.azul.MixedBag;
+
+public class BagTest {
+    @Test
+    public void shouldVerifyThereIsBag() {
+
+        ArrayList<Character> bag = new ArrayList<>();
+        Bag my = new Bag();
+        bag.add('R');
+        assertNotEquals(bag, my.getBag());
+    }
+
+    @Test
+    public void shouldVerifyThereIsMixedBag() {
+
+        ArrayList<Character> mixed = new ArrayList<>();
+        MixedBag mix = new MixedBag();
```

Comment:
Please add a separate `MixedBagTest` file for this one

Reply:
with the deletion of Mixedbag class and making Bag class to handle its methods it is now joined in one Test class